py/server/deephaven/experimental/iceberg.py

```diff
@@ -250,3 +251,80 @@ def adapter_aws_glue(
     except Exception as e:
         raise DHError(e, "Failed to build Iceberg Catalog Adapter") from e
 
+
+def adapter(
+        properties: Dict[str, str],
+        name: Optional[str] = None
+) -> IcebergCatalogAdapter:
+    """
+    Create an Iceberg catalog adapter for an Iceberg catalog created from configuration properties. These properties
+    map to the Iceberg catalog Java API properties and are used to create the catalog and file IO implementations.
```

Comment:
Not clear enough.  If a user comes to this doc, and they don't already know what they want, they probably will not keep reading.